ReactAndroid/src/main/java/com/facebook/react/views/webview/events/TopLoadingErrorEvent.java

```diff
@@ -7,7 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-package com.facebook.catalyst.views.webview.events;
+package com.facebook.react.views.webview.events;
```

Comment:
Thanks for cleaning this, but this is really orthagonal.  Want to put up a quick PR to fix these and then I can accept and then this one will be easier to read?